secure.go

```diff
@@ -466,6 +481,28 @@ func (s *Secure) processRequest(w http.ResponseWriter, r *http.Request) (http.He
 		responseHeader.Set(coopHeader, s.opt.CrossOriginOpenerPolicy)
 	}
 
+	// Cross Origin Resource Policy header.
+	if len(s.opt.CrossOriginResourcePolicy) > 0 {
+		responseHeader.Set(corpHeader, s.opt.CrossOriginResourcePolicy)
+	}
+
+	// Cross-Origin-Embedder-Policy header.
+	if len(s.opt.CrossOriginEmbedderPolicy) > 0 {
+		responseHeader.Set(coepHeader, s.opt.CrossOriginEmbedderPolicy)
+	}
+
+	// X-DNS-Prefetch-Control header.
+	if s.opt.XDNSPrefetchControl {
```

Comment:
Currently with the logic below, everyone using this library will be setting the `X-DNS-Prefetch-Control` header to `off` which we will want to avoid. This should be an opt in header with a length check like the others.

Reply:
Thank you for the review, addressed it here [60e3ee8](https://github.com/unrolled/secure/pull/102/commits/60e3ee83a3f76683c6b8ce638867884d84c5924c) and [9f6ab01](https://github.com/unrolled/secure/pull/102/commits/9f6ab013d8b70a801c9b8210159b96d2417a758b). Let me know what you think.